alf/examples/merlin_dmlab_collect_good_objects.gin

```diff
@@ -0,0 +1,77 @@
+# MERLIN on deepmindlab rooms_collect_good_objects
+
+import alf.algorithms.merlin_algorithm
+import alf.algorithms.decoding_algorithm
+import alf.trainers.on_policy_trainer
+
+# `DeepmindLab` are required,
+#   see `https://github.com/deepmind/lab` to build `DeepmindLab`
```

Comment:
adding more detail for build dmlab data package:
https://github.com/deepmind/lab/blob/master/data/brady_konkle_oliva2008/README.md
python package:
https://github.com/deepmind/lab/blob/master/python/pip_package/README.md